sway/input/seatop_default.c

```diff
@@ -220,7 +220,14 @@ static void handle_tablet_tool_tip(struct sway_seat *seat,
 	struct sway_container *cont = node && node->type == N_CONTAINER ?
 		node->sway_container : NULL;
 
-	if (cont) {
+	if (wlr_surface_is_layer_surface(surface)) {
```

Comment:
Note for reviewer: the mouse button down path first tries to focus an empty workspace. However, we don't need to do so here because if `surface` is null, we'll be hitting the pointer emulation path anyway.